### ISSUE TITLE
`date` bug in MASTER.sh

### ISSUE THREAD

**Issue by IanLee1521** [LLNL/llnl.github.io#55]
It looks like there is an option being used in `MASTER.sh` for the `date` command which macOS doesn't like, I get the following when running the script:

```
date: illegal option -- I
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
```

**Comment by LRWeber** [comment]
Good catch. The `-I` option is just a shortcut for a particular date format.
From the man page:
> Output an ISO 8601 compliant date/time string., '%Y-%m-%d'.

Does `date '+%Y-%m-%d'` work on macOS? If so, that would be a perfectly fine substitute for `date -I`.

**Comment by IanLee1521** [comment]
Looks like it:

```
$ date '+%Y-%m-%d'
2017-09-06
```

**Comment by LRWeber** [comment]
Fixed by #56
